src/shared/common/components/workstack.jsx

```diff
@@ -387,13 +387,13 @@ class WorkstackAllocate extends Component {
             case ColumnRenderer.TRUNCATE_TEXT:
                 return <td key={row.uuid + column.dataValueKey} className='govuk-table__cell govuk-table__cell--truncated' title={value}>{value}</td>;
             case ColumnRenderer.CONTRIBUTIONS_WARNING:
-                if (row.data.CaseContributions) {
-                    const dueContribution = JSON.parse(row.data.CaseContributions)
-                        .filter(contribution => contribution.data && !contribution.data.contributionStatus)
-                        .map(contribution => contribution.data.contributionDueDate)
+                if (row.somu && row.somu.caseContributions) {
+                    const dueContribution = row.somu.caseContributions
+                        .filter(contribution => !JSON.parse(contribution).contributionStatus)
```

Comment:
Could you do a map previous to this to do the `JSON.parse` and then filter afterwards? Means you don't repeat the parse?

Reply:
Yep think it increases readability